osu.Game/Overlays/ChatOverlayV2.cs

```diff
@@ -268,6 +266,8 @@ private void currentChannelChanged(ValueChangedEvent<Channel> channel)
                 return;
             }
 
+            selectorActive.Value = false;
```

Comment:
This doesn't handle the case of switching from a channel to selector, in which `CurrentChannel` is supposed to be null. Causing the following behaviour:

https://user-images.githubusercontent.com/22781491/167482021-6ce99940-be5e-46f9-84fc-7855d7e40f35.mp4

(notice how adding another channel no longer selects it, while the first one does)

The behaviour of selecting a channel when joining it may not be desirable, but that can be fixed by removing it in `ChannelManager.JoinChannel` instead.

Reply:
Is it ok to change the behaviour of the `ChannelManager` like this? I am under the impression that it's current behaviour is relied up in some places?

Reply:
Probably need to check all usages and confirming they won't break as a result of this change.